app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeries.java

```diff
@@ -14,25 +19,53 @@
     private int mDrawerIconResId;
     private int mTitleResId;
     private int mDescriptionResId;
+    private int mDefaultIconResId;
     private int mLogoResId;
-    private int mColorPrimary;
+    private int mSpecialEventTheme;
     private DateTime mStartDateInMillis;
     private DateTime mEndDateInMillis;
 
-    public TaggedEventSeries(String tag, 
+    public TaggedEventSeries(Context context,
+                             @StyleRes int specialEventTheme,
+                             @NonNull String tag,
+                             DateTime startDateInMillis,
+                             DateTime endDateInMillis) {
+
+        final ContextThemeWrapper themeWrapper = new ContextThemeWrapper(context, specialEventTheme);
```

Comment:
I also thought about this. There is no easy way to change the theme of an
activity without having a theme as a style.
Overscroll color, button colors actionbar background. All of these have to
be updated one by one programmatically. At least that's the result of my
research. Please fix me if you know how to.

Also the icons colors texts will not come from resources. They will come
from the API.

So if that is the case, we have to change lots of things here for dynamic
event series.

What do you think?

On Tue, Mar 3, 2015, 5:53 PM Friedger Müffke <notifications@github.com>
wrote:

> In
> app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeries.java
> <https://github.com/gdg-x/frisbee/pull/277#discussion_r25698124>:
>
> >      private DateTime mStartDateInMillis;
> >      private DateTime mEndDateInMillis;
> >
> > -    public TaggedEventSeries(String tag,
> > +    public TaggedEventSeries(Context context,
> > +                             @StyleRes int specialEventTheme,
> > +                             @NonNull String tag,
> > +                             DateTime startDateInMillis,
> > +                             DateTime endDateInMillis) {
> > +
> > +        final ContextThemeWrapper themeWrapper = new ContextThemeWrapper(context, specialEventTheme);
>
> This only works for programmed themes, correct? If we get the data from
> the server we don't need this. Hence it might make more sense to wrap the
> ids in a java object and create this either from the programmed theme or
> from server data.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/277/files#r25698124>.
>